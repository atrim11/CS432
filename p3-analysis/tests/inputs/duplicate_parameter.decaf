def int add(int x, int x) { return x; }  // Error: duplicate parameter 'x'
