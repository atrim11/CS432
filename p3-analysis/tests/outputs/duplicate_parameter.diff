--- outputs/duplicate_parameter.txt	2024-10-25 11:03:54.842827212 -0400
+++ expected/duplicate_parameter.txt	2024-10-25 10:52:35.744531287 -0400
@@ -1,3 +1,2 @@
+Duplicate symbols named 'x' in scope started on line 1
 Program does not contain a 'main' function
-Duplicate parameter names 'x' in function 'add' on line 1
-Duplicate parameter names 'x' in function 'add' on line 1
