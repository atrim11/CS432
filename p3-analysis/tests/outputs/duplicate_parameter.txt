Program does not contain a 'main' function
Duplicate symbols named 'x' in scope started on line 1
