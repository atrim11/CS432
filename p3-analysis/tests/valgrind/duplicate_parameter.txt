==288637== Memcheck, a memory error detector
==288637== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==288637== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==288637== Command: ../decaf inputs/duplicate_parameter.decaf
==288637== 
Program does not contain a 'main' function
Duplicate symbols named 'x' in scope started on line 1
==288637== 
==288637== HEAP SUMMARY:
==288637==     in use at exit: 0 bytes in 0 blocks
==288637==   total heap usage: 1,753 allocs, 1,753 frees, 203,324 bytes allocated
==288637== 
==288637== All heap blocks were freed -- no leaks are possible
==288637== 
==288637== For lists of detected and suppressed errors, rerun with: -s
==288637== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
