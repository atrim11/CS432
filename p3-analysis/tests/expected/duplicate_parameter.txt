Duplicate symbols named 'x' in scope started on line 1
Program does not contain a 'main' function
